dir.targets

```diff
@@ -45,4 +45,15 @@
     <!-- We set this property to avoid MSBuild errors regarding not setting TargetFrameworkProfile (see above line) -->
     <PortableNuGetMode>true</PortableNuGetMode>
   </PropertyGroup>
+
+  <!--
+    add the test-runtime project.json to the list of supplemental test data so it will get included
+    in the test archive; the stress test build system depends on it when building test mixes.
+  -->
+  <ItemGroup Condition="'$(IsTestProject)'=='true'">
```

Comment:
I agree about the path being repo dependent but do we think the notion of stress testing which is a BuildTools concept (I'm assuming) requires this for all repo's? If so we should probably put it into BuildTools and let individual repo's provide the path.